specification/postgresql/resource-manager/Microsoft.DBforPostgreSQL/preview/2017-12-01-preview/postgresql.json

```diff
@@ -1520,6 +1520,115 @@
         "modelAsString": false
       }
     },
+    "MinimalTlsVersion": {
+      "type": "string",
+      "description": "Enforce a minimal Tls version for the server.",
+      "enum": [
+        "TLS1_0",
+        "TLS1_1",
+        "TLS1_2",
+        "TLSEnforcementDisabled"
+      ],
+      "x-ms-enum": {
+        "name": "MinimalTlsVersionEnum",
+        "modelAsString": true
+      }
+    },
+    "ServerPrivateEndpointConnection": {
+      "description": "A private endpoint connection under a server",
+      "type": "object",
+      "properties": {
+        "id": {
+          "description": "Resource ID.",
```

Comment:
same comment

Reply:
Note please apply the previous feedback on these properties to the new spec.  You might consider whether you want to have these properties in a common json file.

Reply:
Resource ID of the Private Endpoint Connection

Reply:
Resource ID of the Private Endpoint Connection. 

Yes @markcowl . I agree that many teams (sql , rdbms, keyvault, storage..) will benefit if ServerPrivateEndpointConnection is a common property. Let me connect with network team and talk about this.